src/commands/hld/reconcile.ts

```diff
@@ -134,16 +170,18 @@ export const reconcileHld = async (
   repositoryName: string,
   absHldPath: string
 ) => {
-  const managedServices = bedrockYaml.services;
-  const managedRings = bedrockYaml.rings;
+  const { services: managedServices, rings: managedRings } = bedrockYaml;
 
   // Create Repository Component if it doesn't exist.
   // In a pipeline, the repository component is the name of the application repository.
-  await dependencies.createRepositoryComponent(
-    dependencies.exec,
-    absHldPath,
-    repositoryName
-  );
+  await dependencies
+    .createRepositoryComponent(dependencies.exec, absHldPath, repositoryName)
+    .catch(err => {
```

Comment:
wondering if we can `.catch` these closer to the invocation of `exec` within createRepositoryComponent, etc - thoughts?